compiler/rustc_passes/src/check_attr.rs

```diff
@@ -455,14 +481,67 @@ impl CheckAttrVisitor<'tcx> {
                 .sess
                 .struct_span_err(
                     meta.span(),
-                    &format!("`#[doc(alias = \"...\")]` is the same as the item's name"),
+                    &format!(
+                        "`#[doc(alias{})]` is the same as the item's name",
+                        if is_list { "(\"...\")" } else { " = \"...\"" },
+                    ),
                 )
                 .emit();
```

Comment:
You use this quite a lot - maybe add a helper function that takes `"is the same as the item's name"` and adds the rest of the boilerplate itself?